server/api/call.go

```diff
@@ -144,8 +146,21 @@ func MakeCall(url string, namevalues ...string) *Call {
 }
 
 func (c *Call) GetValue(name, defaultValue string) string {
```

Comment:
Shouldn't this return an interface and let the caller dig deeper into the value they want to handle?

Right now the `opt["value"].(string)` part seems not very intuitive.

Reply:
@larkox This is a helper for the hello app which can probably be removed. It is basically used for:

- If the form field is a string, give me the string
- If the form field is a select option, give me the `value`

For other fields, `string` may not make sense, so maybe this function shouldn't be here.